Packs/Core/ReleaseNotes/3_0_82.md

```diff
@@ -0,0 +1,14 @@
+
+#### Playbooks
+
+##### New: Unprivileged process opened a registry hive
+
+- This Playbook is designed to handle the 'Unprivileged process opened a registry hive' alerts and executes the following:
+
+Investigation:
+Check the signature of the Actor & CGO processes. 
+Check the prevalence of the Actor & CGO process and Actor & CGO CommandLine
+Check for related XDR alerts using MITRE tactics to identify any malicious activity.
+
+Remediation:
+Based on the signature status of the Actor & CGO processes, prevalence of the Actor & CGO process and Actor & CGO CommandLine, and related alerts if found, the playbook will terminate the causality process if any malicious parameters are found. (Available from Cortex XSOAR 6.10.0).
```

Comment:
The remediation part is like the condition task name that I dont like (I think its unclear) please rephrase this sentence  (also in the playbook description)